plugins/inputs/bind/bind.go

```diff
@@ -0,0 +1,108 @@
+package bind
+
+import (
+	"bufio"
+	"encoding/xml"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Bind struct {
+	Urls                 []string
+	GatherMemoryContexts bool
+	GatherViews          bool
+}
+
+var sampleConfig = `
+  ## An array of BIND XML statistics URI to gather stats.
+  ## Default is "http://localhost:8053/".
+  urls = ["http://localhost:8053/"]
+  gather_memory_contexts = false
+  gather_views = false
+`
+
+var client = &http.Client{
+	Timeout: time.Duration(4 * time.Second),
+}
+
+func (b *Bind) Description() string {
+	return "Read BIND nameserver XML statistics"
+}
+
+func (b *Bind) SampleConfig() string {
+	return sampleConfig
+}
+
+func (b *Bind) Gather(acc telegraf.Accumulator) error {
+	var outerr error
+	var errch = make(chan error)
```

Comment:
We should update this to use acc.AddError, my fault for taking so long to review.